### PR TITLE
Create CCP4I2 citations tab and update the ensemble rmsd calculation

### PR DESCRIPTION
**Pull request by linucks**

### CODE REVIEW

File: ample/ensembler/ensemble_rmsds.py [hunk 1]
```diff
@@ -5,45 +5,76 @@
...
+
+def parse_rmsd_pair(fh):
+    for line in fh:
+        line = line.strip()
+        if not line:
+            continue
+        if 'RMSD             :' in line:
+            rmsd = float(line.split()[2])
+            break
+    assert rmsd
```
Comment (fsimkovic) [inline ample/ensembler/ensemble_rmsds.py:43]: If ``rmsd`` not defined this will break

File: ample/ensembler/ensemble_rmsds.py [hunk 2]
```diff
@@ -5,45 +5,76 @@
...
-                mlist.append([float(r) for r in line.split()[1:]])
-        return list(np.array(mlist).diagonal())
-
-    nmodels = iotbx.pdb.pdb_input(ensemble).construct_hierarchy().models_size()
+    num_models = iotbx.pdb.pdb_input(ensemble).construct_hierarchy().models_size()
      
     cmd = '{} '.format(gesamt_exe)
-    for i in range(nmodels):
+    for i in range(num_models):
         cmd += '{} -s "/{}/" '.format(ensemble, i+1)
```
Comment (fsimkovic) [inline ample/ensembler/ensemble_rmsds.py:75]: `os.sep` or `os.path.join`

File: ample/ensembler/ensemble_rmsds.py [hunk 3]
```diff
@@ -5,45 +5,76 @@
...
+    if mode == 'sheaf':
+        cmd += ' -sheaf-x'
      
     script = './gesamt.sh'
```
Comment (fsimkovic) [inline ample/ensembler/ensemble_rmsds.py:79]: `os.path.join`

**Review by fsimkovic — CHANGES_REQUESTED** [review]
Please address couple of points

File: ample/ensembler/ensemble_rmsds.py (hunk 1 above)
Comment (linucks) [reply ample/ensembler/ensemble_rmsds.py:43]: It's meant to

File: ample/ensembler/ensemble_rmsds.py (hunk 2 above)
Comment (linucks) [reply ample/ensembler/ensemble_rmsds.py:75]: Not a path

File: ample/ensembler/ensemble_rmsds.py (hunk 3 above)
Comment (linucks) [reply ample/ensembler/ensemble_rmsds.py:79]: Not a path

**Review by fsimkovic — APPROVED** [review]

**Pull request merged**